src/app/shared/capitalize-first.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { capitalize } from 'lodash';
+
+/**
+ * Converts the first character of string to upper case and the remaining to lower case.
+ */
+@Pipe({
+  name: 'capitalizeFirst'
+})
+export class CapitalizeFirst implements PipeTransform {
```

Comment:
Given that the only usage of this pipe must be changed, or the text would always be in English, this pipe can be removed. I don't see other usages for it right now.